sources/packages/backend/apps/db-migrations/src/sql/ApplicationExceptionRequests/Add-col-exception-request-status.sql

```diff
@@ -0,0 +1,27 @@
+ALTER TABLE
+    sims.application_exception_requests
+ADD
+    COLUMN exception_request_status sims.application_exception_request_status NOT NULL DEFAULT 'Pending';
+
+COMMENT ON COLUMN sims.application_exception_requests.exception_request_status IS 'Status of the application exception request.';
+
+-- Update the status of existing exception requests which belong to completed exceptions(approved or declined).
+-- If the exception is approved then the status of all the associated exception requests should be set to approved.
+-- If the exception is declined then the status of all the associated exception requests should be set to declined.
+UPDATE
+    sims.application_exception_requests exception_request
+SET
```

Comment:
Should we also set the audit columns while executing this update?

Reply:
I believe it would be better to have the audit columns updated. I can update.